client/src/components/Header/Header.style.js

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export const H1 = styled.h1`
+  color: #fff;
+  font-size: 28px;
+  padding: 0;
+  margin: 0;
+  height: 36px;
+  img {
+    height: 56px;
+  }
+`;
+
```

Comment:
Not sure what is the right thing to do here, but I think it would make sense to put Styled before each of these classes. e.g. StyledH1, StyledUser, StyledBasket. Good to keep it consistent I think.

Reply:
We're only doing this as I can't use header above as the class is called Header and Jem said StyledHeader was common industry practice in that scenario. 

Jem says it would be unnecessary to do this consistently.